reactor-netty-core/src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -91,6 +92,8 @@ public abstract class PooledConnectionProvider<T extends Connection> implements
 	final Duration inactivePoolDisposeInterval;
 	final Duration poolInactivity;
 	final Duration disposeTimeout;
+	final int expectedConnectionPools;
```

Comment:
IMO it is better to call it maximum connection pools. Wdyt?

```suggestion
	final int maxConnectionPools;
```